### PR TITLE
Feature/upgrade preact

### PR DESCRIPTION
**Pull request by stefaniacardenas**
Upgraded Preact to use the latest version. This will allow us to use stateless component without having to impurifying them.

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-181-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json [hunk 1]
```diff
@@ -92,17 +96,12 @@
...
-    "redux": "3.5.2",
-    "supports-webp": "1.0.3",
-    "visibilityjs": "1.2.4",
-    "wpt": "https://github.com/onfido/js-client-tracker/tarball/master",
-    "babel-runtime": "6.23.0",
-    "deep-equal": "1.0.1",
     "reconnectingwebsocket": "1.0.0",
-    "eventemitter2": "2.1.3",
-    "query-string": "4.1.0",
     "redux": "3.4.0",
```
Comment (hjheath) [inline package.json]: Should we be using redux 3.4.0 or 3.5.2?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://181-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json (hunk 1 above)
Comment (stefaniacardenas) [inline package.json]: Why 3.5.2? I guess we could just upgrade to the latest version (3.7.2) unless there is any reason not to.

Comment (hjheath) [inline package.json]: Redux was defined twice in the dependencies. You removed 3.5.2. I'm not sure which version we were actually using, but we should make sure we are still using this version.

Comment (rfreitas) [inline package.json]: we had two redux? we should go with the highest one.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-181-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://181-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**